docs/architecture_documentation/authentication.rst

```diff
@@ -19,15 +19,14 @@ After login the realm has to switched to qunicorn in the top left dropdown menu.
 
 Start Keycloak
 --------------
-The Keycloak Docker Compose: :file:`keycloak-docker-compose.yml` can be started by executing the following command:
+To start Keycloak, run the Docker Compose: :file:`docker-compose.yml` with the following command:
 
 .. code-block:: bash
 
-    docker compose -f keycloak-docker-compose.yaml -p qunicorn-core up -d
+    docker compose -f docker-compose.yaml -p qunicorn-core up -d
```

Comment:
I don't think that `-p qunicorn-core` is required, as we do not use profiles in the docker compose.
(But we could use profiles to not start Keycloak by default and only start it when using the `keycloak` profile...)